content/documentation/rpc/wallet/create.mdx

```diff
@@ -3,6 +3,8 @@ title: wallet/create
 description: RPC Wallet | Iron Fish Documentation
 ---
 
+<Warning>Please use this API instead.</Warning>
```

Comment:
This should be fixed up to point to a particular API before we merge it

Reply:
ah sorry I didn't mean to merge this, removing it now